### DISCUSSION TITLE
20240414-18d8eb1更新后出现的问题

### DISCUSSION THREAD

**Discussion by LLIce** [kkkgo/PaoPaoGateWay#87] [问和答] [answered]
新版对proxy-providers支持是不是出现了问题，同样的配置文件在老版本正常使用，新版本就有问题，删除proxy-providers后才能正常用。

mode=yaml
yaml信息
![LQMAK_6Z$RD8EP@~9USS0 E](https://github.com/kkkgo/PaoPaoGateWay/assets/13741076/2a425375-b248-4b2b-be50-c4209f578b7e)

**Comment by LLIce** [comment]
![XI %FQ)YN1)G(1~~FCACX`1](https://github.com/kkkgo/PaoPaoGateWay/assets/13741076/327141ec-6d7f-4aca-99a7-bcd571a643ef)
这提示是跟fastnode功能有关吗？

**Reply by kkkgo** [reply]
https://github.com/kkkgo/PaoPaoGateWay/blob/18d8eb197d4aa28aabdd569f10c74152abfc0921/ReadMe.md?plain=1#L132

**Reply by kkkgo** [reply, marked as answer]
proxy-providers和ppgw无关，取决于核心解析实现。
如果fast_node=yes将会忽略所有规则和proxy-groups。

**Reply by LLIce** [reply]
fast_node改为no后，可以正常用了。
之前的版本yes状态也能用proxy-provider原来是bug吗。

**Reply by kkkgo** [reply]
可以试试最新发布的镜像，放开了`fast_node`限制，可以兼容你这个情况。
但仍需注意的是，这并不是预期的使用方式，因为你既然自定义了`proxy-groups`，那自然就可以使用`proxy-groups`的`url-test`来自动选择最快节点。此外，`fast_node`被设计用于`GLOBAL`选择，`fast_node`功能的`ext_node`属性就不会按预期工作，当然，`proxy-providers`也自带`filter`功能，所以说`fast_node`这个功能不是预期在手搓配置的情况下用的，他的场景往往是一个简单的`suburl`下使用。

**Reply by LLIce** [reply]
可以了，谢谢作者。

**Comment by JJFJJML** [comment]
兄弟麻烦问一下，你用的哪个版本的核心，还有能不能给个ISO的文件目录树，谢谢你

**Reply by LLIce** [reply]
meta1.18.3
![J)UON5K16JB9)B UTF1E9F0](https://github.com/kkkgo/PaoPaoGateWay/assets/13741076/9ccb3625-88e6-4569-82a0-0b4137859750)

**Reply by JJFJJML** [reply]
谢谢
